.github/workflows/generate_files.yml

```diff
@@ -9,8 +9,12 @@ on:
       # to exclude it.
       - '!21.x'
 
+permissions: {}
```

Comment:
Why not put this in the other workflow files?

Reply:
I see only one workflow without `permissions` - update_php_repo.yml which didn't exist at the moment when the pull-request was created. Or do you mean why it is `read`, but not `{}` in other workflows?

Reply:
Yea I meant why is this the only one where you make the workflow-wide default empty?